### PR TITLE
Submissions Page Scaffold

### PR DESCRIPTION
**Pull request by mendelB**
### What's this PR do?

This pull request adds the basic layout for the campaign submissions page, including the GraphQL query though it's data is yet unused.

### How should this be reviewed?
👀 

### Any background context you want to provide?
Next we'll parse out the primary action, use that to create/populate a new campaign info card, and populate the submissions card for each type of post.

We do have a more specialized version of the recommended campaigns gallery in mind for a v2!

### Relevant tickets

References [Pivotal #178932467](https://www.pivotaltracker.com/story/show/178932467).

### Checklist

- [x] This PR has been added to the relevant Pivotal card.
- [ ] Documentation added for new features/changed endpoints.
- [x] Added screenshots of front-end changes on small, medium, and large screens.
- [ ] Added appropriate feature/unit tests.

Screen Shot 2021-09-20 at 12 31 38
Screen Shot 2021-09-20 at 12 31 47
Screen Shot 2021-09-20 at 12 31 55

### CODE REVIEW

File: resources/assets/components/pages/AccountPage/Submissions/CampaignSubmissions.js [hunk 1]
```diff
@@ -1,25 +1,99 @@
...
   useEffect(() => {
     addBreadcrumb({
-      path: '/us/account/campaigns',
+      path: '/us/account/campaigns/complete',
       title: 'Campaigns',
     });
   }, []);
 
+  const userId = getUserId();
+  const campaignId = match.params.campaignId;
```
Comment (katiecrane) [inline resources/assets/components/pages/AccountPage/Submissions/CampaignSubmissions.js:41]: I haven't seen this used before - cool!

**Review by katiecrane — APPROVED** [review]
Perfect setup for this pass on it!

File: resources/assets/components/pages/AccountPage/Submissions/CampaignSubmissions.js (hunk 1 above)
Comment (mendelB) [reply resources/assets/components/pages/AccountPage/Submissions/CampaignSubmissions.js:41]: Ooh, you know what, you inspired me to do a search, and I feel like this approach might actually be a bit cleaner?
https://github.com/DoSomething/phoenix-next/blob/bf56e9b65b7d3a1977517981b786f1950722c01b/resources/assets/components/App.js#L154

I'll update in the follow-up PR!

Comment (katiecrane) [reply resources/assets/components/pages/AccountPage/Submissions/CampaignSubmissions.js:41]: Ooo yeah I think I prefer that too!

**Pull request merged**